azure_functions_worker/constants.py

```diff
@@ -55,3 +55,8 @@
 
 # Paths
 CUSTOMER_PACKAGES_PATH = "/home/site/wwwroot/.python_packages/lib/site-packages"
+
+# Flag to index functions in handle init request
+PYTHON_ENABLE_INIT_INDEXING = "ENABLE_INIT_INDEXING"
```

Comment:
If it's a python specific app setting, we can add it to app_setting_manager.py and we can log it

Reply:
yup. added